libs/html-tag/.browserslistrc

```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+last 1 Chrome version
```

Comment:
Question: I wonder why only 1 version ?

Reply:
That's what shcematics generate now for libs and apps.
I guess since all browsers auto-update now you only need to worry about the latest version =)